source/api_documentation/sms_messaging_api/index.rst

```diff
@@ -0,0 +1,144 @@
+====================
+SMS Messaging API
+====================
+
+.. raw:: html
+
+  <div class="alert alert-warning">
+    <b>Notice:</b>
+    The SMS Messaging API is in Alpha. Please contact your CSM or the support team for more information.
+  </div>
+
+------------------------------------
+Overview
+------------------------------------
+The SMS Messaging API provides the following functionality:
+
+- manage SMS-enabled phone numbers
+- send SMS messages
+- retrieve SMS messages and their associated Delivery Reports
+
+------------------------------------
+Authentication and Authorization
```

Comment:
I would call out here that, unlike _other_ Invoca API, SMS Messaging will _only_ support an `Authorization` "Bearer" header, no `GET` or `POST` parameter named `oauth_token`.

Reply:
That's a good point. 👍

Reply:
Updated.